src/components/auth/login/Recaptcha/Recaptcha.web.js

```diff
@@ -1,5 +1,5 @@
 import React, { forwardRef, useImperativeHandle, useRef } from 'react'
-import ReCAPTCHA from 'react-google-recaptcha'
+import GoogleRecaptcha from 'react-google-recaptcha'
```

Comment:
fixes lint issue. it seems the updated eslint have added some case-insensitive check for the variables names, so it warns ReCAPTCHA and Recaptcha are similar